doc/tests/check-nix-code-blocks.nix

```
@@ -0,0 +1,27 @@
+{
+  runCommand,
+  markdown-code-runner,
+  nixfmt-rfc-style,
+}:
+
+runCommand "manual_check-nix-code-blocks"
+  {
+    nativeBuildInputs = [
+      markdown-code-runner
+      nixfmt-rfc-style
+    ];
+  }
+  ''
+    set +e
+
+    mdcr --check --config ${./mdcr-config.toml} ${./..}
+
+    if [ $? -ne 0 ]; then
+      echo -e "Error: mdcr command failed. Please make sure the Nix code snippets in Markdown files are correctly formatted.\n"
+      echo -e "Run this command from the root path to fix: mdcr --log debug --config doc/tests/mdcr-config.toml doc/\n"
```

```suggestion
      echo -e "Run this command from the Nixpkgs repository root for automatic formatting: mdcr --log debug --config doc/tests/mdcr-config.toml doc/\n"
```

This is just for the wording. I recommend letting the command stand out visually, so it shows like

```console
blablabla:

    command foo bar
```

---

Done.